lib/logstash/outputs/dynatrace.rb

```diff
@@ -160,6 +160,10 @@ def log_retryable_response(response)
         end
       end
 
+      def log_partial_success_response(response)
+        @logger.error("Encountered partial success response", code: response.code, body: response.body)
```

Comment:
you're logging an error here instead of a warning like PR title suggests

Reply:
I wasn't sure whether it should be error or warn. I think I'll go with warn because I think error implies some sort of exception or _unexpected_ failure (network failure, out of memory, etc). Warning means telling the user something is happening they may not like, but is expected with the inputs given. I'll update this